ingest/data_library/dataloading.sh

```diff
@@ -1,12 +1,6 @@
 #!/bin/bash
 source ingest/data_library/config.sh
 
-import_csv lpc_historic_district_areas
-import_csv doi_evictions
-import_csv dcp_access_subway_sbs
-import_csv dcp_access_ada_subway
-import_csv hpd_hny_units_by_building
-import_csv dcp_housing
 for year in {2010..2020}
```

Comment:
@td928 with all these files removed, what is the point of having the `dataloading.sh` ?

Reply:
The dot traffic injuries we still import the old way. FacDB also has a combination of dataloading the traditional way and dataloading with `datasets.yml`

Reply:
now it is reduced to only files not following our one file one version convention namely the `dcp_dot_trafficinjuries` which pull all 10 years of data. Also might be in there is the geographies or other datasets we don't necessarily want to consider versioning.